client/src/components/Common/DelayedInput.vue

```diff
@@ -2,6 +2,7 @@
     <b-input-group>
         <b-input
             class="search-query"
+            id="tool-input"
```

Comment:
This id will be overly specific with overlap.  What you can do instead is switch this to use vue $refs -- https://vuejs.org/guide/essentials/template-refs.html

Reply:
Ah i see, I felt weird adding the id too. I'll use $refs.